public/components/add-modules-data/guides/office.js

```diff
@@ -0,0 +1,181 @@
+/*
+* Wazuh app - Amazon Web Services interactive extension guide
+* Copyright (C) 2015-2021 Wazuh, Inc.
+*
+* This program is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+*
+* Find more information about this on the LICENSE file.
+*/
+export default {
+  id: 'office',
+  name: 'Office 365',
+  wodle_name: 'office',
+  description: 'Configuration options of the Office 365 wodle.',
```

Comment:
**comment**: This file is used for a not-released feature. At the moment, maybe this could not be necessary. But I guess, there is not a problem if we add to the source code.